### DISCUSSION TITLE
Genoype calling for NAM population/allotetraploid with autotetraploid reference

### DISCUSSION THREAD

**Discussion by PaulaEB** [lvclark/polyRAD#32] [Q&A] [answered]
Hello! I am new to polyRAD and I would like to know how can I set the parents in a multiparenting population (was an adapted NAM pop) in which I have 4 pollen recipients and 1 recurrent pollen donor. Also, if the recurrent pollen donor is autotetraploid (and also this was the reference used for the alignments) but the 4 pollen recipients are allotetraploids how can I set those

**Comment by lvclark** [comment, marked as answer]
Hi!  Right now polyRAD is not set up for multiparent populations.  It is something I have been wanting to do (see #18) but I have not had an example dataset to use for testing and benchmarking.  If you are interested in sharing your dataset (maybe a subset with 1000-5000 markers) with me it is something that I could work on.  However, it would probably take me at least several months of development since I can only work on polyRAD in my free time now and I have a lot going on in my personal life at the moment.

It sounds like you would set the whole population as being autotetraploid. Generally I think of allotetraploids as having sets of chromosomes that will definitely not pair with each other at meiosis, but if you are making recurrent crosses with an autotetraploid, they must be pairing.

**Reply by PaulaEB** [reply]
Thank you very much Lindsay! So it would be ok treat each half-sib family as a biparental pop?

**Reply by lvclark** [reply]
Yes, that should work!
